### PR TITLE
FAQ - update 'mobile_data_costs' entry for #971

### PR DESCRIPTION
**Pull request by heinezen**
Changelog:
- Chang english sentence from plural to singular
- "Datenvolumina" -> "Datenvolumen"
- sight rephrasing of sentences

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -694,7 +694,7 @@
                         "anchor": "mobile_data_costs",
                         "active": true,
                         "textblock": [
-                            "The German network operators have committed themselves to a so-called zero rating. This means that the mobile data volume for the use of the Corona-Warn-App is provided free of charge both in Germany and within the EU. Mobile data that are transmitted via Corona-Warn-App when diagnosis keys are uploaded or when the server notifies you in case of exposure is free of charge and will not be deducted from your data plan.",
+                            "The German network operators have committed themselves to a so-called zero rating. This means that the mobile data volume for the use of the Corona-Warn-App is provided free of charge both in Germany and within the EU. Mobile data that is transmitted via Corona-Warn-App when diagnosis keys are uploaded or when the server notifies you in case of exposure is free of charge and will not be deducted from your data plan.",
```
Comment (MikeMcC399) [inline src/data/faq.json:697]: "via Corona-Warn-App" =>
"via the Corona-Warn-App"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
One minor suggestion for you.

File: src/data/faq.json (hunk 1 above)
Comment (heinezen) [reply src/data/faq.json:697]: fixed :)

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
left new - right old 
image

image


@heinezen @MikeMcC399 Thanks! LGTM!

**Pull request merged**